src/styles/utils/variables.scss

```
@@ -0,0 +1,6 @@
+$background-color-card: #FFF;
+$background-color-header: #FFF;
+$background-button-color: #00ACDC;
+$color-header: #00ACDC;
```

Why did you create two variables with the same values?
```suggestion
$background-color-card: #FFF;
$color-blue: #00ACDC;
```